libs/langchain/langchain/output_parsers/boolean.py

```diff
@@ -19,24 +21,26 @@ def parse(self, text: str) -> bool:
             boolean
 
         """
-        cleaned_upper_text = text.strip().upper()
```

Comment:
I believe adding .split() is sufficient to make the required change?

```python
cleaned_upper_text = text.strip().upper().split()
```

Reply:
Not really, as `'My anwser is "YES"'` will `.split()` into `[..., '"YES"']`. Meanwhile the regex will remove the surrounding quotes.